### ISSUE TITLE
Change the "Could not find device ..." to more meaningful

### ISSUE THREAD

**Issue by martonmiklos** [psmay/pk2cmd#4]
If a device does not found in the device database it outputs "Could not find device ...", which is a bit misleading because at first I was associated that I have HW problem. 
I propose to change it something like:
"Could not find device in the device database: "

This is just a kind of "note issue to myself". Let me know if you agree and I will fix it with a PR.

**Comment by mikebdp2** [comment]
@martonmiklos It's sad your pull requests weren't addressed at this abandoned repo; could be a good idea to restore them just to increase the visibility of your forked repo. Thank you for the improvements!

**Comment by martonmiklos** [comment]
@mikebdp2 Yeah I did some cleanup on my github and other stuff to get a clear view on my TODO lists. 
Leaving PRs open makes sense due to the visibility terms, so I have reopened them.
